docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 dask_cuda
 numpydoc==1.1.0
 sphinx_click==2.7.1
-sphinx_rtd_theme==0.5.1
\ No newline at end of file
+sphinx_rtd_theme==0.5.1
+tomli
```

Comment:
It was needed because the docs also did a build. Or at least that was true when the docs here were built on ReadTheDocs. However docs are now built by RAPIDS.

Plus we can do the same simplification that you noted elsewhere of dropping the requirements file and including these in `pyproject.toml`. So have since made that change.